modules/040-node-manager/images/machine-controller-manager/werf.inc.yaml

```diff
@@ -1,40 +1,26 @@
 image: {{ .ModuleName }}/{{ .ImageName }}
-from: {{ .Images.BASE_ALPINE }}
+fromImage: common/distroless
 docker:
   ENV:
     WORKDIR: /
     ENTRYPOINT: "/machine-controller-manager"
 import:
 - artifact: {{ .ModuleName }}/{{ .ImageName }}/build
-  add: /src/bin/machine-controller-manager
+  add: /src/machine-controller-manager
   to: /machine-controller-manager
   before: setup
 ---
 artifact: {{ .ModuleName }}/{{ .ImageName }}/build
-from: {{ .Images.BASE_GOLANG_ALPINE }}
-fromCacheVersion: "20210607"
-git:
-- url: https://github.com/deckhouse/mcm.git
-  to: /src
-  tag: v0.36.0-flant.13
-  stageDependencies:
-    install:
-    - '**/*'
+from: {{ .Images.BASE_GOLANG_20_ALPINE }}
 mount:
 - fromPath: ~/go-pkg-cache
   to: /go/pkg
-ansible:
-  beforeInstall:
-  - name: "Install dependencies"
-    apk:
-      name:
-      - mercurial
-      - make
-      - bash
-      - tzdata
-      update_cache: yes
+shell:
   install:
-  - name: "Build"
-    shell: make build-local
-    args:
-      chdir: /src
+  - apk add git
+  setup:
+  - mkdir /src && cd /src
+  - git clone --depth 1 --branch v0.36.0-flant.13 {{ env "SOURCE_REPO" | default "https://github.com" }}/deckhouse/mcm.git .
```

Comment:
env? Mabe we will use werf variable as here 
https://github.com/deckhouse/deckhouse/pull/5520/files#diff-af1683f87626a34a29eed0b462e74aa9c802c76b5242f8070dd96afc135df6d7R32
?

Reply:
Fixed.